types/heroku-logger/index.d.ts

```diff
@@ -12,28 +13,26 @@
 
  =============================================== */
 
-export function trace(message: string, data?: object): void;
-export function debug(message: string, data?: object): void;
-export function info(message: string, data?: object): void;
-export function warn(message: string, data?: object): void;
-export function error(message: string, data?: object): void;
-export function fatal(message: string, data?: object): void;
+ export function trace(message: string, data?: object): void;
+ export function debug(message: string, data?: object): void;
+ export function info(message: string, data?: object): void;
+ export function warn(message: string, data?: object): void;
+ export function error(message: string, data?: object): void;
+ export function fatal(message: string, data?: object): void;
```

Comment:
Looks like white space was accidentally added here

Reply:
Thanks, I think it has been fixed, could you check it as well?

Reply:
Yup, looks much better, thanks for making the fixes! (original and white space)

Reply:
Thanks to you for creating this! :)